private/prepare_freq_matrices.m

```diff
@@ -106,11 +106,11 @@
     ft_error('the requested reference channel is not found in the data');
   end
   if any(strncmp(tok, 'rpt', 3))
```

Comment:
idem, see above

Reply:
done for this function. I suspect that for a lot of older code in the repo this could be done (i.e. not tokenizing de dimord anymore, but use contains and startsWith